integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/steps/Hooks.java

```diff
@@ -72,10 +74,9 @@ public void cleanUp() {
                                     p -> ProcessorResource.deleteProcessor(token, bridgeId,
                                             p.getId()));
                             Awaitility.await()
-                                    .atMost(Duration.ofMinutes(2))
+                                    .atMost(Duration.ofMinutes(4))
                                     .pollInterval(Duration.ofSeconds(5))
-                                    .until(() -> ProcessorResource.getProcessorList(token, bridgeId)
-                                            .getSize() == 0);
+                                    .untilAsserted(() -> assertThat(ProcessorResource.getProcessorList(token, bridgeId).getItems()).isEmpty());
```

Comment:
Could we please change to form:
```
assertThat(ProcessorResource.getProcessorList(token, bridgeId).getItems()).as("explaining the fail").isEmpty()
```

Something like `processor with <id> still present`